app/models/user.rb

```diff
@@ -10,6 +10,10 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: :author_id, dependent: :destroy
   has_many :likes, foreign_key: :author_id, dependent: :destroy
 
+  def is?(role)
```

Comment:
👏👏👏

Reply:
Hi @brytebee , thank you for the review. I'm a bit confused with this. Is there anything I need to change or fix with this `is?` action?